www_src/blocks/generator.js

```diff
@@ -11,18 +12,19 @@ Generator.IMAGE = "image";
 
 Generator.generateBlock = function(options) {
```

Comment:
Personally I would rather get rid of this function altogether and just export the block components